src/mvi/gpgmm_d3d12.cpp

```diff
@@ -75,13 +75,19 @@ namespace gpgmm::d3d12 {
     // static
     HRESULT Heap::CreateHeap(const HEAP_DESC& descriptor,
```

Comment:
Any changes to this (exported) function also need to be made in `include/gpgmm_d3d12.h` or it won't compile.

Reply:
Yup!